app/views/initiatives/index.html.erb

```diff
@@ -11,7 +11,11 @@
           <p class="init-description"><%= initiative.description %></p>
           <div>
             <div class="init-footer w-row">
-              <div class="w-col w-col-2" style="padding: 0;"><%= image_tag "icon-calendar.svg", class: "icon-target" %></div>
+              <% if initiative.completion %>
```

Comment:
There might be an opportunity here to not use an if-else if the icon name could be derived from the state of the initiative. Then you could use string interpolation within `image_tag`. Something like `<%= image_tag "#{state_icon}.svg" ... %>` or similar. Not required, but something to consider.

Reply:
We've altered the if else statement with a ternary operator